### PR TITLE
Benji/ios 495 delivery type update

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: iOS/Flows/Conversation/Cells/CenterConversationDetailView.swift [hunk 1]
```diff
@@ -52,6 +52,8 @@ class CenterConversationDetailView: UICollectionReusableView, ConversationUIStat
         self.addSubview(self.rightLabel)
         self.addSubview(self.imageView)
         self.imageView.contentMode = .scaleAspectFit
+        
+        self.leftLabel.setText("No messages")
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Cells/CenterConversationDetailView.swift:56]: added for the case of no messages

File: iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift [hunk 2]
```diff
@@ -298,6 +298,7 @@ class ConversationListViewController: ViewController, ConversationListCollection
 
                 guard !Task.isCancelled else { return }
 
+                self?.messageInputAccessoryView.resetDeliveryType()
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift:301]: we reset the delivery type only when switching convos

File: iOS/UI/Swipeable Input/DeliveryTypeView.swift [hunk 3]
```diff
@@ -33,61 +32,59 @@ class DeliveryTypeView: BaseView {
...
-                              image: MessageContext.timeSensitive.image,
-                              identifier: nil,
-                              discoverabilityTitle: nil,
-                              attributes: [],
-                              state: urgentState) { [unowned self] _ in
-            self.didSelectContext?(.timeSensitive)
-            self.configure(for: .timeSensitive)
+        
+        var actions: [UIAction] = []
+        MessageContext.allCases.forEach { value in
```
Comment (Benjamindodgson) [inline iOS/UI/Swipeable Input/DeliveryTypeView.swift:69]: way cleaner

**Review by MartinJibber — APPROVED** [review]
Cool

**Pull request merged**